pkgs/by-name/fl/flashgbx/package.nix

```
@@ -0,0 +1,62 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+, copyDesktopItems
+, makeDesktopItem
+, qt6
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "flashgbx";
+  version = "3.36";
+
+  src = fetchFromGitHub {
+    repo = "FlashGBX";
+    owner = "lesserkuma";
+    rev = version;
+    sha256 = "sha256-MbeQKdmAGt4bTSleU1Dlve7fLDfldS2gIYUCIbgDAk4=";
```

```suggestion
    hash = "sha256-MbeQKdmAGt4bTSleU1Dlve7fLDfldS2gIYUCIbgDAk4=";
```

this is a small nitpick, but i'm pretty sure the `hash` attribute is preferred these days.

---

Okay, I'm happy to use the preferred nomenclature.